src/main/java/org/owasp/benchmark/score/parsers/AcunetixReader.java

```diff
@@ -26,23 +26,66 @@
 public class AcunetixReader extends Reader {
 	
 	public TestResults parse( Node root ) throws Exception {
-        TestResults tr = new TestResults( "Acunetix WVS", true, TestResults.ToolType.DAST);
-        Node scan = getNamedChild( "Scan", root );
 
-//        <ScanGroup ExportedOn="11/9/2015, 21:42">
-//        <Scan>
-//         <Name><![CDATA[Scan Thread 1 ( https://172.16.11.1:8443/benchmark/ )]]></Name>
-//         <ShortName><![CDATA[Scan Thread 1]]></ShortName>
-//         <StartURL><![CDATA[https://172.16.11.1:8443/benchmark/]]></StartURL>
-//         <StartTime><![CDATA[11/9/2015, 14:50:33]]></StartTime>
-//         <FinishTime><![CDATA[11/9/2015, 15:31:02]]></FinishTime>
-//         <ScanTime><![CDATA[40 minutes, 29 seconds]]></ScanTime>
+      if (root.getNodeName().equalsIgnoreCase("acunetix-360")) {
+
+	/* This is for the 2020 format that looks like:
+		 <acunetix-360 generated="28/02/2020 08:59 AM">
+			<target>
+				<scan-id>f55650d3326f40f30d92ab6e04af1794</scan-id>
+				<url>https://localhost:8443/benchmark/</url>
+				<initiated>27/02/2020 04:49 PM</initiated>
+				<duration>01:57:21.2094646</duration>
+			</target>
+			<vulnerabilities>
+				<vulnerability>
+					<LookupId>cf5644f1-31ee-4092-0e15-ab6e04af7f51</LookupId>
+					...
+	*/
+        TestResults tr = new TestResults( "Acunetix 360", true, TestResults.ToolType.DAST);
+        
+        Node target = getNamedChild( "target", root );
+        String duration = getNamedChild("duration", target ).getTextContent();
+        // duration format is: 01:57:21.2094646
+        tr.setTime( duration.substring(0, duration.lastIndexOf('.')));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If the lastIndexOf() search on line 50 does not find the substring and returns -1, the call to substring() on line 50 will throw IndexOutOfBoundsException.